spec/tbd_tests_spec.rb

```diff
@@ -1566,3 +1566,414 @@
+        },
+        "logs": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Log"
+          }
+        }
+      },
+      "additionalProperties": false,
+      "definitions": {
+        "PSI": {
+          "description": "Set of PSI-values (in W/K per m) for thermal bridges",
+          "type": "object",
+          "properties": {
+            "id": {
+              "title": "Unique PSI set identifier",
+              "type": "string"
+            },
+            "rimjoist": {
+              "title": "Floor/wall edge PSI",
+              "type": "number",
+              "minimum": 0
+            },
+            "parapet": {
+              "title": "Roof/wall or exposed-floor/wall edge PSI",
+              "type": "number",
+              "minimum": 0
+            },
+            "fenestration": {
+              "title": "Window, door, skylight perimeter PSI",
+              "type": "number",
+              "minimum": 0
+            },
+            "concave": {
+              "title": "Wall corner [0°,135°) PSI",
+              "type": "number",
+              "minimum": 0
+            },
+            "convex": {
+              "title": "Wall corner (225°,360°] PSI",
+              "type": "number",
+              "minimum": 0
+            },
+            "balcony": {
+              "title": "Floor/balcony edge PSI ",
+              "type": "number",
+              "minimum": 0
+            },
+            "party": {
+              "title": "Party wall edge PSI",
+              "type": "number",
+              "minimum": 0
+            },
+            "grade": {
+              "title": "Floor/foundation edge PSI",
+              "type": "number",
+              "minimum": 0
+            }
+          },
+          "additionalProperties": false,
+          "required": [
+            "id"
+          ],
+          "minProperties": 2
+        },
+        "KHI": {
+          "description": "KHI-value (in W/K) for a point thermal bridge",
+          "type": "object",
+          "properties": {
+            "id": {
+              "title": "Unique KHI identifier",
+              "type": "string"
+            },
+            "point": {
+              "title": "Point KHI-value",
+              "type": "number",
+              "minimum": 0
+            }
+          },
+          "additionalProperties": false,
+          "required": [
+            "id",
+            "point"
+          ]
+        },
+        "Edge": {
+          "description": "Surface(s) edge as thermal bridge",
+          "type": "object",
+          "properties": {
+            "psi": {
+              "title": "PSI-set identifier",
+              "type": "string"
+            },
+            "type": {
+              "title": "PSI-set type, e.g. 'parapet'",
+              "type": "string"
+            },
+            "length": {
+              "title": "Edge length (m), 10cm min",
+              "type": "number",
+              "minimum": 0.1
```

Comment:
I'd make minimum be 0, you can ignore them later but you will have edges that are small

Reply:
The Topolys tolerances should filter out any edge under 1 cm (and we've flirted with moving this up to an inch). But you're right, as Topolys can/will generate 2-inch or 3-inch edges depending on OSM geometry. It's bad design on the user's part (or any third-party app), but that's irrelevant. Agree to drop to "0", but will likely add an "exclusiveMinimum" property - we should never come across a strict "0" value.